src/util/saveArticles.ts

```diff
@@ -104,7 +108,7 @@ async function downloadBulk(listOfArguments: any[], downloader: Downloader): Pro
             argList,
             async (arg) => {
                 const resp: any = {};
-                resp.path = arg.val.path;
+                resp.path = shouldConvertImageUrlToWebp(arg.val.path, downloader.webp) ? arg.val.path + '.webp' : arg.val.path;
```

Comment:
This should happen in the optimization part: only one test and then (1) make the optim (2) change the mime-type and filename in the `resp`.